src/components/Layout/ExternalLinkMenuItem.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Icon } from '@mdi/react';
+
+type ExternalLinkMenuItemProps = {
+  url: string;
+  name: string;
+  icon: string;
+};
+
+export const ExternalLinkMenuItem = React.memo(({
```

Comment:
Make the components default exports. Hot reload doesn't work otherwise.

Reply:
didn't notice it did that as I used extract